### PR TITLE
Update Contract's page

### LINKED ISSUE #140: Correct contracts' definition and update contract address
### Description
The description of OptimismMintableERC20Factory contract on Ethereum(L1) and and Ethereum Sepolia (L1) specified in https://docs.lisk.com/contracts is wrong

> It is responsible for creating ERC20 contracts on L2 that can be used for depositing native L1 tokens into.

It should rather be

> It is responsible for creating ERC20 contracts on L1 that can be used for depositing native L2 tokens into.

Also the contract address of OptimismMintableERC20Factory contract on Ethereum(L1) `0xD00e38514d66bf1B761a8937559c6b2854A5B3ad` points to the implementation but other OptimismMintableERC20Factory contract addresses, including Ethereum Sepolia (L1), rather point to the proxy. So the address should be `0xc1dA06CC5DD5cE23bABa924463de7F762039252d`

### Location
contracts.md

### PR DESCRIPTION
**Pull request by TalhaMaliktz**
### What was the problem?

This PR resolves #140

### CODE REVIEW

File: docs/building-on-lisk/contracts.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ The following L1 contracts have been deployed to the **Ethereum Sepolia Testnet*
...
 | **OptimismPortal**                | [0xe3d90F21490686Ec7eF37BE788E02dfC12787264](https://sepolia.etherscan.io/address/0xe3d90F21490686Ec7eF37BE788E02dfC12787264) | It is the main entry point to deposit funds from L1 to L2. It also allows proving and finalizing withdrawals. |
 | **ProtocolVersions**              | [0x1Bb17EB31bDdFd30F63D4FAe6c8Eb85D9e9b1f48](https://sepolia.etherscan.io/address/0x1Bb17EB31bDdFd30F63D4FAe6c8Eb85D9e9b1f48) | It is used to manage superchain protocol version information. |
-| **OptimismMintableERC20Factory**  | [0x269d632C1E518a922C30C749cFD3f82Eb5C779B0](https://sepolia.etherscan.io/address/0x269d632C1E518a922C30C749cFD3f82Eb5C779B0) | It is responsible for creating ERC20 contracts on L2 that can be used for depositing native L1 tokens into. |
+| **OptimismMintableERC20Factory**  | [0x269d632C1E518a922C30C749cFD3f82Eb5C779B0](https://sepolia.etherscan.io/address/0x269d632C1E518a922C30C749cFD3f82Eb5C779B0) | It is responsible for creating ERC20 contracts on L1 that can be used for depositing native L2 tokens into. |
```
Comment (Tschakki) [inline docs/building-on-lisk/contracts.md:85]: This contract is still no proxy, should this be updated as well?

**Review by Tschakki — CHANGES_REQUESTED** [review]

File: docs/building-on-lisk/contracts.md (hunk 1 above)
Comment (TalhaMaliktz) [reply docs/building-on-lisk/contracts.md:85]: It is proxy! Please see: https://sepolia.etherscan.io/address/0x269d632C1E518a922C30C749cFD3f82Eb5C779B0#code

Comment (Tschakki) [reply docs/building-on-lisk/contracts.md:85]: Ah ok, I was confused because of the missing "Read as Proxy" & "Write as Proxy" tabs.

**Review by Tschakki — APPROVED** [review]
LGTM

**Review by Incede — APPROVED** [review]

**Pull request merged**